Source/Core/DolphinQt/SkylanderPortal/SkylanderPortalWindow.cpp

```diff
@@ -0,0 +1,860 @@
+// Copyright 2022 Dolphin Emulator Project
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+// DolphinQt code copied and modified for Dolphin from the RPCS3 Qt utility for Creating, Loading
```

Comment:
You might need to copy the RPCS3 licence header, not just credit the original code.

Reply:
I couldn't find the RPCS3 licence headers within their codebase, maybe I didn't look hard enough